tests/e2e/backup_restore_suite_test.go

```diff
@@ -227,7 +227,7 @@ func runBackup(brCase BackupRestoreCase, backupName string) bool {
 	return nsRequiresResticDCWorkaround
 }
 
-func runRestore(brCase BackupRestoreCase, backupName, restoreName string, nsRequiresResticDCWorkaround bool) {
+func runRestore(brCase BackupRestoreCase, backupName, restoreName string, nsRequiresResticDCWorkaround bool, expectedErr error) {
```

Comment:
to avoid changes in backup_restore_suite_test.go and in must-gather_suite_test.go, I would copy this function to virt_backup_restore_suite_test.go with a TODO to be removed after problem is fixed (this is planed to be fixed in on our sprints, right?)

Reply:
Okay, I tried this out in the latest push. The changes are definitely more contained now.